### DISCUSSION TITLE
Benq display(EW 2780Q) can not adjust the volume of the built-in monitor sound.

### DISCUSSION THREAD

**Discussion by hongyoudan** [MonitorControl/MonitorControl#1267] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

Benq display(EW 2780Q) can not adjust the volume of the built-in monitor sound, although the slider has changed, but the actual volume does not change. 
But the brightness of the display is normally adjustable.

**Comment by hongyoudan** [comment]
DDC is enabled.

**Comment by waydabber** [comment]
Sadly many monitors do not support volume controls via DDC. There is nothing much we can do about it I am afraid. You can also check if you are use USB-C for control as DDC volume control does not work via the HDMI port of your Mac (if it has any).
